jme3-android-examples/jme3-android-tests/jbullet/build/.transforms/753141cfd132cfc6b6d425f62db41945.bin

```diff
@@ -0,0 +1 @@
+o/jetified-jbullet.jar
```

Comment:
what is jetified-jbullet?

Reply:
the Jetifier is a standalone tool shipped with the new Androidx packages, it eases migration of libraries to be usable inside androidx,

https://developer.android.com/studio/command-line/jetifier

Reply:
Can't we include jbullet and stack-alloc with Android Gradle plugin without using the Jetifier?

Reply:
I have tried, but there was an error, cannot remember.

Reply:
Yes, it works & i was wrong about using jetifier in this situation, i replied back on forums.